### DISCUSSION TITLE
Intel RVP board isn't booting after flashing

### DISCUSSION THREAD

**Discussion by leenaarora18** [zephyrproject-rtos/zephyr#82502] [Q&A]
Issue with Intel Raptor Lake and MEC1521 SPI Image Flashing:

I'm working with Intel Raptor Lake and using the open-source ECFW Zephyr (https://github.com/intel/ecfw-zephyr) to generate the ksc.bin file. Here’s what I did:

Cloned the CPGZephyrDocs repository and added the necessary SPI image generation files.

Set the environment variable for everglades_spi_gen_RomE.

Successfully built the project using west build -c -p auto -b mec1501_mtl_p, which generated the ksc.bin file.

Stitched the ksc.bin with Intel BIOS and flashed it using Dediprogrammer onto the Intel Raptor Lake S board.

After flashing, both the CPU fan and ATX fan started working, and the power sequence voltages were active, but the system did not boot. However, these signals (CPU fan, ATX fan, and power sequence) only activate when I manually touch the RSMRST signal using a multimeter.

Any insights on why the system isn't booting after flashing?

**Comment by kwd-doodling** [comment]
@nashif Do you know who can help for RVP board booting?

**Comment by kartben** [comment]
Not sure what board target `mec1501_mtl_p` is, but it looks like it's a downstream board? Will convert to a discussion for now in case someone is able to answer your question.
